crates/oxc_minifier/src/ast_passes/peephole_remove_dead_code.rs

```diff
@@ -379,17 +443,16 @@ mod test {
     fn test_array_literal() {
         fold("([])", "");
         fold("([1])", "");
-        // fold("([a])", "a");
-        // fold("([foo()])", "foo()");
+        fold("([a])", "a");
```

Comment:
Ci currently fails on this one - i remove the `a` but closure keeps it
closures behaviour doesn't make sense to me here - on L455, closure removes `b` ?

Reply:
ignore this - looks like the tests were ported over incorrectly:

https://github.com/google/closure-compiler/blob/f888b14e237f6f18f9cb66eeef3e79f398c45a68/test/com/google/javascript/jscomp/PeepholeRemoveDeadCodeTest.java#L1477-L1479